test/test_state_update.py

```diff
@@ -26,11 +26,16 @@ def run_before_and_after_test():
     # after test
     devnet_proc.kill()
 
-def get_state_update(block_hash = None):
+def get_state_update(block_hash = None, block_number = None):
```

Comment:
You can remove the spaces around `=`

Reply:
[ab4dbc3](https://github.com/Shard-Labs/starknet-devnet/pull/49/commits/ab4dbc38d9169eed0129fc67199dd292f8c700cd)